markdown/index.md

```diff
@@ -4,9 +4,8 @@ Nav:   home.nav
 Bedrock Linux
 =============
 
-~+Bedrock Linux~x is a meta Linux distribution which allows users to
-mix-and-match components from other, typically incompatible distributions.
-~+Bedrock~x integrates these components into one largely cohesive system.
+~+Bedrock Linux~x allows to run a linux system that is composed of different linux distributions.
+It integrates typically incompatible components into one largely cohesive system.
```

Comment:
I find this to be less comprehensible and less accurate description than what it is replacing.